CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,131 @@
+# Contributor Covenant Code of Conduct
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5624734)